packages/accounts-base/globals_client.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @namespace Accounts
+ * @summary The namespace for all client-side accounts-related methods.
+ */
+Accounts = new AccountsClient();
```

Comment:
It's important to define the new `Accounts` instance after all the code setting up its `.prototype` has finished running, otherwise we might call methods from the constructor that haven't been defined yet.